client/blocks/like-button/button.jsx

```diff
@@ -49,6 +50,9 @@ class LikeButton extends PureComponent {
 
 	toggleLiked( event ) {
 		if ( ! this.props.isLoggedIn ) {
+			if ( this.props.onLoggedOut ) {
```

Comment:
nit: I think a more common naming pattern would be to call the event `onToggleLiked` or similar and then perform the logged in check within that function

Reply:
Tidied up this logic (I think!) to do the login check in the `ReaderLikeButton.onLikeToggle` function.